.travis.yml

```diff
@@ -42,16 +51,19 @@ jobs:
     script:
     - scripts/setup-minikube-for-linux.sh
     - make test CHARTS=enterprise-suite-latest ES_TEST=frontend
+
   - name: Reactive-sandbox test
     script:
     - make test CHARTS=reactive-sandbox
+
   - stage: deploy
     name: Deploy to GCS
     # Only deploy for pushes to master (and if manually invoked)
     if: type IN (push, api)
     script:
     # Decrypt credentials for GCS system account helmchart@es-repo.iam.gserviceaccount.com
-    - openssl aes-256-cbc -K $encrypted_f01ffbb90c44_key -iv $encrypted_f01ffbb90c44_iv -in resources/es-repo-7c1fefe17951.json.enc -out resources/es-repo-7c1fefe17951.json -d
+    - mkdir -p /tmp/resources
```

Comment:
I was considering using `$TMPDIR` instead of `/tmp` for something else.  Probably doesn't matter but was curious what you thought.

Reply:
Not too worried about portability here since this is travis specific, so we can rely on the build environment. (Famous last words?)